### PR TITLE
Add gwtc tools package

### PR DESCRIPTION
**Pull request by chadhanna**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by chadhanna** [comment]
@conda-forge-admin, please ping team

**Comment by chadhanna** [comment]
@conda-forge/help-python

File: recipes/gwtc-tools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gwtc_tools-{{ version }}.tar.gz
+  sha256: 6672e0a735f816ca7ee230ea92e641e02e5f614f062e6532eccb03f5137e3b42
+
+build:
+  entry_points:
+    - gwtc_update_pipeline_gevents = gwtc:gwtc_update_pipeline_gevents
+    - gwtc_diff = gwtc:gwtc_diff
+    - gwtc_get_gevent_coinc_files = gwtc:gwtc_get_gevent_coinc_files
+    - gwtc_create_from_query = gwtc:gwtc_create_from_query
```
Comment (duncanmmacleod) [inline recipes/gwtc-tools/meta.yaml:17]: The `gwtc_test` executable is missing from this list. I think this is responsible for the failure.

```suggestion
    - gwtc_update_pipeline_gevents = gwtc:gwtc_update_pipeline_gevents
    - gwtc_diff = gwtc:gwtc_diff
    - gwtc_get_gevent_coinc_files = gwtc:gwtc_get_gevent_coinc_files
    - gwtc_create_from_query = gwtc:gwtc_create_from_query
    - gwtc_test = gwtc:gwtc_test
```

Comment (Rhiannon-Udall) [reply recipes/gwtc-tools/meta.yaml:17]: @chadhanna I've implemented this suggestion in this PR: https://github.com/chadhanna/staged-recipes/pull/1

**Comment by Rhiannon-Udall** [comment]
@conda-forge/help-python this package should now be ready for review; please let us know if there's anything else we can do to help with the process!

**Review by xhochy — APPROVED** [review]

**Pull request merged**